dom/observable/tentative/observable-ctor.any.js

```diff
@@ -0,0 +1,685 @@
+test(() => {
+  assert_true(
+    typeof Observable === "function",
+    "Observable constructor is defined"
+  );
+}, "Observable constructor");
+
+test(() => {
+  let initializerCalled = false;
+  const results = [];
+
+  const source = new Observable((subscriber) => {
+    initializerCalled = true;
+    subscriber.next(1);
+    subscriber.next(2);
+    subscriber.next(3);
+    subscriber.complete();
+  });
+
+  assert_false(
+    initializerCalled,
+    "initializer should not be called by construction"
+  );
+
+  source.subscribe({
+    next: (x) => results.push(x),
+    error: () => assert_unreached("should not be called"),
```

Comment:
The "should not be called" is pretty unspecific. In this case it's not too bad, because there's only one of these. But in some of the cases below we have the same assertion message for multiple handlers, so in general I'd prefer to make these mention which handler they correspond to (here and below). So `assert_unreached("error handler should not be called")`, etc.

Reply:
I've gone ahead and fixed this